packages/ai-ide/src/browser/summarize-session-command-contribution.ts

```diff
@@ -37,7 +42,38 @@ export class SummarizeSessionCommandContribution implements CommandContribution
     @inject(CoderAgent)
     protected readonly coderAgent: CoderAgent;
 
+    @inject(ArchitectTaskSummaryAgent)
+    protected readonly architectTaskSummaryAgent: ArchitectTaskSummaryAgent;
```

Comment:
Is this Agent actually used?

Reply:
good catch! Removed it in the follow-up commit